CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ List of incompatible ABI changes in this release:
   memory allocation failures.
 * avifReadImage(), avifJPEGRead() and avifPNGRead() now remove the trailing zero
   byte from read XMP chunks, if any. See avifImageFixXMP().
+* The 'mode' member of the avifImageMirror struct was renamed 'axis'.
```

Comment:
I didn't mention this change in the list of incompatible ABI changes because even though this is an incompatible API change, this doesn't change the ABI. I wonder we should change that list to be the list of incompatible API changes.

Reply:
You can also start an "API changes" list.